app/decorators/contribution_decorator.rb

```diff
@@ -0,0 +1,19 @@
+class ContributionDecorator < Draper::Decorator
+  delegate_all
+
+  def created_at_formatted
+    model.created_at.strftime("%d/%m/%Y")
+  end
+
+  def reviewed_by_short_name
+    return '' if reviewed_by.nil?
+
+    reviewed_by.first_and_last_name.split.map(&:first).join.upcase
+  end
+
+  def reviewed_at_formatted
+    return '' if reviewed_at.nil?
+
+    model.reviewed_at.strftime("%d/%m/%Y")
```

Comment:
You can also do something like `model.reviewed_at.to_fs(:date)`, it is actually a better approach once it'll be using the convention defined for the entire project, look [here](https://github.com/Codeminer42/Punchclock/blob/master/config/initializers/date_time_formats.rb#L3).